src/main/scala/com/kifi/franz/FakeSQSQueue.scala

```diff
@@ -17,7 +17,7 @@ trait FakeSQSQueue[T] extends SQSQueue[T] {
 
   override def initQueueUrl(): String = ""
 
-  override def send(msg: T, messageAttributes: Option[Map[String,String]]): Future[MessageId] = Future.successful(MessageId(""))
+  override def send(msg: T, messageAttributes: Option[Map[String,String]], delay: Option[Int]): Future[MessageId] = Future.successful(MessageId(""))
```

Comment:
Would you mind making this default to `None` as well to preserve signature compatibility?